gulpfile.js

```diff
@@ -83,10 +94,19 @@ gulp.task( 'styles', function () {
       log: true
     } ) )
     .pipe( minifyCSS() )
-    .pipe( gulp.dest( 'public/' ) );
+    .pipe( rev() )
+    .pipe( gulp.dest( 'public/' ) )
+    .pipe( gutil.buffer( function ( err, dataFiles ) {
+      gulp.src( files.rev )
+        .pipe( jsoneditor( {
+          'styles': dataFiles.map( function ( dataFile ) {
+            return dataFile.path.replace( dataFile.base, '' ).slice( 0, -4 ).split( '-' )[ 1 ];
+          } ).join( '' )
+        } ) )
+        .pipe( gulp.dest( 'public/' ) );
+    } ) );
```

Comment:
I'm not quite sure about this one, but I think the stream might be returned be the function in `gutil.buffer`.